app/src/main/java/ch/hikemate/app/ui/map/ZoomMapButton.kt

```diff
@@ -23,12 +34,40 @@ const val ZOOM_OUT_BUTTON = "ZoomOutButton"
  */
 @Composable
 fun ZoomMapButton(onZoomIn: () -> Unit, onZoomOut: () -> Unit, modifier: Modifier = Modifier) {
-  Column(modifier = modifier.testTag(ZOOM_MAP_BUTTON)) {
-    Button(onClick = onZoomIn, modifier = Modifier.testTag(ZOOM_IN_BUTTON)) {
-      Icon(imageVector = Icons.Filled.Add, contentDescription = "Zoom In")
-    }
-    Button(onClick = onZoomOut, modifier = Modifier.testTag(ZOOM_OUT_BUTTON)) {
-      Icon(imageVector = Icons.Filled.Remove, contentDescription = "Zoom Out")
-    }
+  Row(modifier = modifier.testTag(ZoomMapButton.ZOOM_MAP_BUTTON).padding(horizontal = 5.dp)) {
+    IconButton(
+        onClick = onZoomOut,
+        modifier =
+            Modifier.testTag(ZoomMapButton.ZOOM_OUT_BUTTON)
+                .size(
+                    width = ZoomMapButton.ZOOM_BUTTON_WIDTH_SIZE,
+                    height = ZoomMapButton.ZOOM_BUTTON_HEIGHT_SIZE)
+                .padding(horizontal = 1.dp),
+        colors =
+            IconButtonDefaults.iconButtonColors(
+                containerColor = MaterialTheme.colorScheme.surface)) {
+          Icon(
+              imageVector = Icons.Filled.Remove,
+              contentDescription = stringResource(id = R.string.zoom_out),
+              tint = MaterialTheme.colorScheme.onSurface,
+              modifier = Modifier)
+        }
+    IconButton(
+        onClick = onZoomIn,
+        modifier =
+            Modifier.testTag(ZoomMapButton.ZOOM_IN_BUTTON)
+                .size(
+                    width = ZoomMapButton.ZOOM_BUTTON_WIDTH_SIZE,
+                    height = ZoomMapButton.ZOOM_BUTTON_HEIGHT_SIZE)
```

Comment:
Question: Same as for the search button, do we need to hardcode the size of the button? For the zoom buttons, unlike the search button, I noticed a change in width when removing the `.size()` modifier. Maybe we can simply keep the width and leave the height free of constraints?

Reply:
I don't think I really understand your question. You are suggesting I remove only the hard-coded height in the size?

Reply:
I removed it.

Reply:
I now remember why I did it. It is because this way it is always allined with the BottomSheet padding. So if you a way to fix that without hardcoding the size let me know.

Reply:
I changed it. Now only the width is hard-coded for it to resemble the one on Figma.